integration-tests/bats/primary-key-changes.bats

```diff
@@ -617,7 +617,7 @@ SQL
     dolt sql -q "alter table t drop primary key"
     run dolt sql -q "show create table t"
     [ $status -eq 0 ]
-    [[ "$output" =~ "NOT NULL" ]] || false
+    [[ "$output" =~ "`pk` int NOT NULL" ]] || false
```

Comment:
Should have a test for a table with two primary key columns and additional non-primary key columns as well